public/js/components/Info.js

```diff
@@ -0,0 +1,154 @@
+const React = require("react");
+const { connect } = require("react-redux");
+const classnames = require("classnames");
+const { getTabs } = require("../selectors");
+
+require("./Info.css");
+const { DOM: dom } = React;
+const ImPropTypes = require("react-immutable-proptypes");
+
+const githubUrl = "https://github.com/devtools-html/debugger.html/blob/master";
+
+function getTabsByBrowser(tabs, browser) {
+  return tabs.valueSeq()
+    .filter(tab => tab.get("browser") == browser);
+}
+
+const Info = React.createClass({
+  propTypes: {
+    tabs: ImPropTypes.map.isRequired
+  },
+
+  displayName: "Info",
+
+  getInitialState() {
+    return {
+      selectedPane: "Firefox"
+    };
+  },
+
+  renderTabs(tabTitle, tabs, paramName) {
+    if (!tabs || tabs.count() == 0) {
+      return null;
+    }
+
+    return dom.div(
+      { className: "tab-group" },
+      dom.ul(
+        { className: "tab-list" },
+        tabs.valueSeq().map(tab => dom.li(
+          { "className": "tab",
+            "key": tab.get("id"),
+            "onClick": () => {
+              window.location = "/?" + paramName + "=" + tab.get("id");
+            } },
+          dom.div({ className: "tab-title" }, tab.get("title")),
+          dom.div({ className: "tab-url" }, tab.get("url"))
+        ))
+      )
+    );
+  },
+
+  renderFirefoxPanel() {
```

Comment:
true, I created a `firstTimeMessage` function which handles a lot of the duplication. The rest is still there as I think there might be more divergence over time.